example-code/snippets/src/backend/ExampleCode.test.ts

```diff
@@ -121,6 +121,55 @@ describe("Example Code", () => {
 
   });
 
+  it("export elements from local bim file", async () => {
+    // edit these values to run
+    const inBimFilePathName: string = ""; // e.g., "c:\\tmp\\foo.bim"
+    const outFBFileNameBase: string = ""; // Optional, for export to FlatBuffer, e.g., "c:\\tmp\\bar"
+    const elementIdArray: Id64Array = []; // Optional, e.g., ["0x2000000000c", "0x2000000000a"]
```

Comment:
Wrap the published code in a function that takes these three values as parameters.